cmd/object-handlers.go

```diff
@@ -711,16 +711,16 @@ func getCpObjMetadataFromHeader(ctx context.Context, r *http.Request, userMeta m
 }
 
 // getRemoteInstanceTransport contains a singleton roundtripper.
-var getRemoteInstanceTransport *http.Transport
-var getRemoteInstanceTransportLongTO *http.Transport
-var getRemoteInstanceTransportOnce sync.Once
+var (
+	getRemoteInstanceTransport     *http.Transport
+	getRemoteInstanceTransportOnce sync.Once
+)
 
 // Returns a minio-go Client configured to access remote host described by destDNSRecord
 // Applicable only in a federated deployment
 var getRemoteInstanceClient = func(r *http.Request, host string) (*miniogo.Core, error) {
 	getRemoteInstanceTransportOnce.Do(func() {
-		getRemoteInstanceTransport = NewGatewayHTTPTransport()
-		getRemoteInstanceTransportLongTO = newGatewayHTTPTransport(time.Hour)
+		getRemoteInstanceTransport = newGatewayHTTPTransport(2 * time.Hour)
```

Comment:
Is this the value you wanted to make configurable?

Reply:
Yes it is configured in config-current.go @klauspost

Reply:
@harshavardhana Shouldn't it be read from there, then? This is a constant.

Reply:
Correct I added it here for safety @klauspost since it uses a really high value

Reply:
Let me think If I can remove and change it further.